cmd/gen-manifests/main.go

```diff
@@ -387,6 +388,17 @@ func mockDepsolve(packageSets map[string][]rpmmd.PackageSet, repos []rpmmd.RepoC
 				specSet = append(specSet, spec)
 			}
 		}
+
+		// generate pseudo packages for the repos
+		for _, repo := range repos {
+			checksum := fmt.Sprintf("%x", sha256.Sum256([]byte(repo.Name)))
+			specSet = append(specSet, rpmmd.PackageSpec{
+				Name:           "repo:" + repo.Name,
+				RemoteLocation: repo.BaseURLs[0] + "/repo:" + repo.Name,
```

Comment:
we should probably chop the `-20240901` (i.e. anything after the last `-`) to add some stability.